pyproject.toml

```diff
@@ -9,10 +9,12 @@ python = "^3.9"
 google-cloud-pubsub = "^2.9.0"
 tinydb = "^4.6.1"
 pytest = "^7.0.0"
+jsonschema = "^4.4.0"
 
 [tool.pytest.ini_options]
 testpaths = [
-    "tests/pubsub.py"
```

Comment:
Can we not just have a wildcard here to match "tests/*"?

Reply:
This was a bad idea spawning from me not wanting to run the GCP reliant tests. I've just gone with the easy route, remove this and just not added a "test_" prefix to the GCP tests file, shoulda done that in the first place :smile: